tests/nightwatch/static-suite-spec.js

```diff
@@ -1,7 +1,9 @@
+const snapshot = require('../../src/nightwatch/snapshot');
+
 module.exports = {
   'Runs the test suite correctly statically': (browser) => {
     browser
       .url(browser.launchUrl)
-      .expect.element('.test').text.to.equal('Test');
+      .perform(snapshot);
```

Comment:
not using custom commands because due to the way they load I can't easily determine the filepath from where the test is running.

Reply:
Ah I left a comment about this up above, I see the issue is around how we set `testDirectory`. You might check with the value of `__dir` is and see if that can be used in any way to determine where `snapshotDirectory` should be.

Reply:
That will give the directory of where the `snapshot.js` file is. I tried a custom command and printed out the nightwatch API object to identify any possible way to figure it out and there is simply nothing there.